lib/daru/index/index.rb

```diff
@@ -222,6 +223,10 @@ def reorder(new_order)
       self.class.new(new_order.map { |i| from[i] })
     end
 
+    def set_name _name
```

Comment:
1. In Ruby, we are never using `set_foo` methods, we name them `foo=` (and if it just sets corresponding variable, it can be defined with `attr_writer`)
2. Method arg names are started with underscore _only_ when you want to show that argument is not used, which is not the case.